### DISCUSSION TITLE
Using the structurizr renderer for diagrams

### DISCUSSION THREAD

**Discussion by panbanda** [avisi-cloud/structurizr-site-generatr#420] [Q&A] [answered]
We have worked hard to use themes in setting up the structurizr DSL and find it quite helpful especially when dealing with AWS and kubernetes deployment views.  I was wondering if there was any way to render those views using the structurizr renderer?

**Comment by jp7677** [comment, marked as answer]
It should nowadays work pretty well. Please see https://github.com/avisi-cloud/structurizr-site-generatr/discussions/306

**Reply by panbanda** [reply]
Well that was easy ha. Thanks
